test/resources/oc.resource

```diff
@@ -43,14 +43,22 @@ Oc Exec
 
 Oc Wait For
     [Documentation]    Runs 'oc wait'.
-    ...    ${type_name}    expects a "type/name", e.g. pod/foo
+    ...    ${kind}    specifies the API kind (pod, cm, etc)
```

Comment:
Like we discussed in the team meeting today, we want test code to be linear and easy to read. That's going to mean duplication, which is OK in this case.

Instead of making this keyword take arguments that lead to if/then statements and error handling, could we just have different keywords for waiting for different things? I could imagine a name like "Pod With Label Should Be Ready" or "Pod With Name Should Be Ready" that takes just the identifying information for the pod and builds the `oc wait` command in exactly 1 way.